### PR TITLE
Move RTCRtpEncodingParameters.maxFramerate to webrtc-pc

### PR DESCRIPTION
**Pull request by Orphis**
It has been implemented in 3 UAs and deserves a promotion.

***
<a href="https://pr-preview.s3.amazonaws.com/Orphis/webrtc-pc/pull/2785.html" title="Last updated on Oct 13, 2022, 5:28 PM UTC (6769e94)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2785/d183990...Orphis:6769e94.html" title="Last updated on Oct 13, 2022, 5:28 PM UTC (6769e94)">Diff</a>

### CODE REVIEW

**Review by aboba — COMMENTED** [review]
Can you add a link to the WPT test?

**Comment by aboba** [comment]
Also I think you need to modify amendments.json

**Comment by dontcallmedom** [comment]
@Orphis I'll take care of the amendments.json change since that one is a bit complicated

File: webrtc.html [hunk 1]
```diff
@@ -8097,6 +8097,17 @@ <h2>
                           exception/throw =] a {{RangeError}}.
                         </p>
                       </li>
+                      <li>
+                        <p>
+                          Verify that the value of each
+                          {{RTCRtpEncodingParameters/maxFramerate}
+                          member in <var>sendEncodings</var> that is defined
```
Comment (jan-ivar) [inline webrtc.html:8104]:
```suggestion
                          member in <var>sendEncodings</var> that is present
```

File: webrtc.html [hunk 2]
```diff
@@ -8838,6 +8849,17 @@ <h2>
                           exception/created =] {{RangeError}}.
                         </p>
                       </li>
+                      <li>
+                        <p>
+                          Verify that each encoding in <var>encodings</var> has
+                          a {{RTCRtpEncodingParameters/maxFramerate}
```
Comment (youennf) [inline webrtc.html:8855]:
```suggestion
                          a {{RTCRtpEncodingParameters/maxFramerate}}
```

**Comment by alvestrand** [comment]
@Orphis will work with @dontcallmedom to fix amendments.json

File: webrtc.html (hunk 1 above)
Comment (Orphis) [reply webrtc.html:8104]: The text for `scaleResolutionDownBy` uses `defined` and I'll keep that for consistency, feel free to update both if you want later though.

File: webrtc.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ async function listAmendments() {
       throw new Error(`Amendments in container with id ${id} are mixing "modification" and "append" difftypes`);
     }
     if (amendments[id].some(a => a.type !== amendments[id][0].type)) {
-      throw new Error(`Amendments in container with id ${id} are mixing "corrections" and "addition" types`);
+      //throw new Error(`Amendments in container with id ${id} are mixing "corrections" and "addition" types`);
```
Comment (Orphis) [inline webrtc.js:88]: @dontcallmedom : Do you want to merge this as is with the commented code?
Feel free to update if needed and then merge the PR.

Comment (dontcallmedom) [reply webrtc.js:88]: yes, this is intended, will merge

**Pull request merged**